src/System.Transactions.Local/src/System/Transactions/TransactionInterop.cs

```diff
@@ -195,7 +195,7 @@ public static Transaction GetTransactionFromTransmitterPropagationToken(byte[] p
 
             // Extract the transaction guid from the propagation token to see if we already have a
             // transaction object for the transaction.
-            byte[] guidByteArray = new byte[16];
+            Span<byte> guidByteArray = stackalloc byte[16];
```

Comment:
Similar comment as above, but:

```cs
Guid txId = new Guid(propagationToken.AsSpan(8, 16));
```

Reply:
Addressed in https://github.com/dotnet/corefx/pull/36838/commits/677b924f461fbcdb6366f45d7844cbc3cbafd6db